src/views/projectExplorer/library.ts

```diff
@@ -76,6 +77,12 @@ export default class Library extends ProjectExplorerTreeItem {
     if (![LibraryType.systemLibrary, LibraryType.defaultUserLibrary].includes(libraryType) && !variable) {
```

Comment:
This if statement and what it contains can be removed since it is now combined with the below checks for setting the `contextValue`.